### PR TITLE
fresh install fix

### PR DESCRIPTION
**Pull request by carsonwright**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,36 +51,32 @@
...
     "source-map-loader": "^0.1.5",
     "style-loader": "^0.13.0",
     "url-loader": "^0.5.7",
-    "webpack": "^1.13.2",
+    "v8-lazy-parse-webpack-plugin": "^0.3.0",
+    "webpack": "1.14.0",
     "webpack-dev-server": "^1.15.0"
   },
   "dependencies": {
-    "color": "^1.0.3",
```
Comment (tomprats) [inline package.json]: I use color for darkening the background

File: src/index.js [hunk 2]
```diff
@@ -2,13 +2,29 @@
 // import "isomorphic-fetch";
 import { h, render } from "preact";
 import "./style";
-import TraitifyClient from "traitify-js-browser-client";
+import Qwest from "qwest";
+Qwest.setDefaultOptions({dataType: 'json', responseType: 'json', cache:true});
 
 let Main = require("./components/main").default;
-var Traitify = TraitifyClient || {};
+class Traitify {
```
Comment (tomprats) [inline src/index.js]: How come you moved from TraitifyClient?

Comment (carsonwright) [inline src/index.js]: Its what was breaking it

File: package.json (hunk 1 above)
Comment (carsonwright) [inline package.json]: I'll add color back in

**Pull request merged**